pygeppetto_server/consumers.py

```diff
@@ -2,26 +2,56 @@
 
 
 def ws_connect(message):
-    print("Socket opened")
+    print('Socket opened')
     # Accept the connection request
-    message.reply_channel.send({"accept": True})
+    message.reply_channel.send({'accept': True})
 
     # 1 -> Send the connection
-    message.reply_channel.send(
-        {"text": json.dumps({"type": "client_id", "data": "{\"clientID\":\"Connection1\"}"})})
+    message.reply_channel.send({
+            'text': json.dumps({
+                'type': 'client_id',
+                'data': {
+                    'clientID': 'Connection1'
```

Comment:
@NeZanyat I think you are breaking the websocket communication because of this change. Can you please have a look at your javascript console and double check you are not getting error? Remember to clear the cache. I think by removing the slash + inverted commas the message is not properly parsed js side.

Reply:
This applies for every single message transfer...